tests/integration_tests/test_experiment.py

```diff
@@ -848,6 +848,31 @@ def on_resume_training(self, is_coordinator):
         )
 
 
+@pytest.mark.distributed
+def test_tabnet_with_batch_size_1(tmpdir, ray_cluster_5cpu):
```

Comment:
Need to be consistnet with the ray cluster used in the module, which should be `ray_cluster_4cpu`.